doc/requirements.txt

```diff
@@ -0,0 +1,61 @@
+#
+# This file is autogenerated by pip-compile with python 3.10
+# To update, run:
+#
+#    pip-compile docs/requirements.in
```

Comment:
Where does this come from? A `requirements.in` does not exist.

Reply:
yes true, i just copied the requirements.txt over from elco's damagescanner. Is there an easy way to auto-generate a requirements.txt file for the documentation from scratch? I guess not?

Reply:
Yes, it says it in this comment: You can create a `requirements.in` that simply states the top-level packages you need. `pip-compile` can then create a `requirements.txt` from that, where all versions are pinned (which keeps your documentation stable). I am guessing you need

```
# requirements.in
ipykernel
nbsphinx
sphinx
sphinx-rtd-theme
```

Reply:
ah oke, it wasn't clear to me how to do / with what to fill this requirements.in. I'll try that

Reply:
made the .in file and compiled the `requirements.txt` file accordingly